src/Raven.Server/Documents/DocumentsStorage.cs

```diff
@@ -1844,6 +1844,22 @@ public static LazyStringValue TableValueToId(JsonOperationContext context, int i
             var ptr = tvr.Read(index, out int size);
             return Slice.From(context.Allocator, ptr, size, ByteStringType.Immutable, out slice);
         }
+
+        public string GetDocumentIDFrom(DocumentsOperationContext context, string changeVector)
```

Comment:
This is not a valid scenario.
You will not be able to find a document that was replicated from another instance

Reply:
IIRC, @maximburyak did a lot of stuff around that when we were building replication.